### ISSUE TITLE
Document how to capture unhandled promise rejections

### ISSUE THREAD

**Issue by benvinegar** [getsentry/raven-node#193]
The raven-node documentation should show how this is possible, probably under [Usage](https://docs.sentry.io/hosted/clients/node/usage/).

Refs #144

**Comment by benvinegar** [comment]
Note that raven-js already documents this [here](https://docs.sentry.io/hosted/clients/javascript/usage/#promises). Update to reflect Node's event hook via `process.on('unhandledRejection')` ([docs](https://nodejs.org/api/process.html#process_event_unhandledrejection)).

**Comment by LewisJEllis** [comment]
I'm all for saving users from boilerplate, so how about adding an option to `client.patchGlobal()` to allow for something like:
```javascript
client.patchGlobal({ unhandledRejections: true });
```
We can even let configuration specify one, the other, or both of `uncaughtException` and `unhandledRejection`; it's just a matter of choosing which event listeners to register.
```javascript
client.patchGlobal({
  uncaughtExceptions: true,
  unhandledRejections: true
});
```

**Comment by benvinegar** [comment]
👍

... although I'm not sure why anyone would want `uncaughtExceptions: false, unhandledRejections: true`. I'd just add `unhandledRejections` unless I'm missing something.

**Comment by rentzsch** [comment]
@benvinegar has handling of `unhandledRejection` landed yet? A grep of master suggests not yet?

**Comment by LewisJEllis** [comment]
No, it will land in 1.0.0, a beta of which will be out this week.

**Comment by rentzsch** [comment]
Heh, I didn't even notice raven-node wasn't marked as a 1.0.0 yet. Sounds great, thanks.

**Comment by LewisJEllis** [comment]
Addressed by #219, closing.
